Oswaldo Vazquez Aparicio/30 Agosto/hackerrank/javasort/Student.java

```diff
@@ -0,0 +1,53 @@
+package hackerrank.javasort;
+
+class Student implements Comparable<Student> {
+    private final Integer id;
+    private final String fname;
+    private final Double cgpa;
+
+    public Student(Integer id, String fname, Double cgpa) {
+        super();
+        this.id = id;
+        this.fname = fname;
+        this.cgpa = cgpa;
+    }
+
+    public Integer getId() {
+        return id;
+    }
+
+    public String getFname() {
+        return fname;
+    }
+
+    public Double getCgpa() {
+        return cgpa;
+    }
+
+    @Override
+    public int compareTo(Student student) {
+        int compareCgpa = cgpa.compareTo(student.getCgpa());
+
+        if (!areEquals(compareCgpa)) {
```

Comment:
OMG!!!
I think you can refactor compareTo() so it would be more readable right?
areEquals() and  invertOrder() are really necessary??? really?